model-test/src/main/java/org/jboss/as/model/test/ModelTestOperationValidatorFilter.java

```diff
@@ -46,13 +46,23 @@
     private final List<OperationEntry> entries;
 
     private ModelTestOperationValidatorFilter(List<OperationEntry> entries) {
-        this.entries = entries;
+        this.entries = createStandardEntries(entries);
         validateNone = false;
     }
 
     private ModelTestOperationValidatorFilter(boolean validateNone) {
         this.validateNone = validateNone;
-        entries = null;
+        entries = validateNone ? null : createStandardEntries(null);
+    }
+
+    private static List<OperationEntry> createStandardEntries(List<OperationEntry> provided) {
+        if (provided == null) {
+            provided = new ArrayList<>();
+        }
+        // Don't check the private internal op that deregisters itself before we get a chance to validate it
+        OperationEntry oe = new OperationEntry(PathAddress.EMPTY_ADDRESS, "boottime-controller-initializer-step", Action.NOCHECK, null);
+        provided.add(oe);
```

Comment:
An alternative to this is to change this OSH to have it not deregister itself and instead just throw an ISE if invoked post-boot. It's private so it's not visible to end users regardless.